c7n/output.py

```diff
@@ -61,6 +61,12 @@ def __init__(self, ctx, namespace=DEFAULT_NAMESPACE):
         self.namespace = namespace
         self.buf = []
 
+    def get_timestamp(self):
+        if os.getenv("C7N_METRICS_TZ", '').upper() in ('TRUE', ''):
```

Comment:
can we comment that this is a backwards compatibility flag, and reference the original issue.